WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -270,11 +270,15 @@ public static void addOrUpdatePost(ReaderPost post) {
     }
 
     public static ReaderPost getPost(long blogId, long postId, boolean excludeTextColumn) {
+        return getPost(false, blogId, postId, excludeTextColumn);
+    }
 
+    public static ReaderPost getPost(boolean isFeed, long blogOrFeedId, long postOrItemId, boolean excludeTextColumn) {
```

Comment:
I would prefer to see this as two separate functions, perhaps `getBlogPost()` and `getFeedPost()`. We could keep this function as-is, but mark it private and have those two new functions rely on it.